app/views/users/curation.html.haml

```diff
@@ -10,7 +10,34 @@
           $( "#reject_user_id" ).val( ui.item.user_id );
         }
       } );
+      $( "#site-admins .dropdown-menu" ).on( "click", function( e ) {
+        e.stopPropagation( );
+      } );
+      $( "#site-admins input[type='checkbox']" ).on( "change", function( e ) {
+        $.ajax({
+          type: $( this ).is( ":checked" ) ? "POST" : "DELETE",
+          url: "/site_admins",
+          data: {
+            site_admin: {
+              user_id: #{ @display_user.id },
```

Comment:
/users/curation is supposed to work without a user_id, so this causes a problem.

Reply:
Thanks - this should be fixed with the latest commit